corev_apu/fpga/xilinx/xlnx_axi_clock_converter/tcl/run.tcl

```diff
@@ -8,10 +8,10 @@ set_property board_part $boardName [current_project]
 
 create_ip -name axi_clock_converter -vendor xilinx.com -library ip -module_name $ipName
 
-set_property -dict [list CONFIG.ADDR_WIDTH {64} CONFIG.DATA_WIDTH {64} CONFIG.ID_WIDTH {5}] [get_ips $ipName]
+set_property -dict [list CONFIG.ADDR_WIDTH {64} CONFIG.DATA_WIDTH {64} CONFIG.ID_WIDTH {9}] [get_ips $ipName]
```

Comment:
Where does the large ID width come from?

Reply:
Aha. We use this script directly to generate the Xilinx IPs instantiated in [culsans_xilinx.sv](https://github.com/planvtech/culsans/blob/pulp/rtl/src/culsans_xilinx.sv). The [CCU](https://github.com/planvtech/ace/blob/master/doc/ace_ccu_top.md) adds log2(NB_CORES) + log2(NB_CORES+1) = 1 + 2, and the [AXI_LLC](https://github.com/pulp-platform/axi_llc#axi-llc) adds one more bit.

The culsans synthesis flow should have its own script for generating the IPs and not modify this one, but it's a too large change to do right now, can we keep this for now and revert it in a later PR?

Reply:
I see. The problem is that this probably breaks the standalone FPGA synthesis (`ariane_xilinx`)

Reply:
Understood. I have reverted our changes now.